src/app/core/data/comcol-data.service.spec.ts

```diff
@@ -162,11 +162,12 @@ describe('ComColDataService', () => {
     });
 
     describe('if the scope Community can\'t be found', () => {
-      it('should throw an error', () => {
+      it('should throw an error', (done) => {
         const result = service.getBrowseEndpoint(options);
         const expected = cold('--#-', undefined, new Error(`The Community with scope ${scopeID} couldn't be retrieved`));
 
         expect(result).toBeObservable(expected);
+        done();
```

Comment:
I think this change is a bit strange. `done` should only be useful when it's used in a callback function for an async call. So I don't think it's doing anything here, in the main body of the `it` function. I think the result would be exactly the same if it was left out.

Reply:
I removed the done and added a take(1) to observable to avoid the issue in the afterAll